docs/contributing-to-airbyte/gradle-cheatsheet.md

```diff
@@ -12,7 +12,10 @@ We have 3 ways of slicing our builds:
 
 In our CI we run **Build Platform** and **Build Connectors Base**. Then separately, on a regular cadence, we build each connector and run its integration tests.
 
-We split Build Platform and Build Connectors Base from each other for a few reasons: 1. The tech stacks are very different. The Platform is almost entirely Java. Because of differing needs around separating environments, the Platform build can be optimized separately from the Connectors one. 2. We want to the iteration cycles of people working on connectors or the platform faster _and_ independent. e.g. Before this change someone working on a Platform feature needs to run formatting on the entire codebase \(including connectors\). This led to a lot of cosmetic build failures that obfuscated actually problems. Ideally a failure on the connectors side should not block progress on the platform side. 3. The lifecycles are different. One can safely release the Platform even if parts of Connectors Base is failing \(and vice versa\).
+We split Build Platform and Build Connectors Base from each other for a few reasons:
+1. The tech stacks are very different. The Platform is almost entirely Java. Because of differing needs around separating environments, the Platform build can be optimized separately from the Connectors one.
```

Comment:
technically in markdown there needs to be an empty line between the bullet list and the paragraph before it I think? 
```suggestion

1. The tech stacks are very different. The Platform is almost entirely Java. Because of differing needs around separating environments, the Platform build can be optimized separately from the Connectors one.
```

Reply:
Unless somehow the markdown viewer on IntelliJ is incorrect I don't _think_ there needs to be an empty line between bulleted list and paragraph

![Screen Shot 2022-08-13 at 22 39 57](https://user-images.githubusercontent.com/108309971/184524129-46c1801d-5f87-4dbc-95de-1ef540863bc3.png)

Based on this [bulleted list tutorial](https://riptutorial.com/markdown/example/2218/bulleted-lists) it doesn't seem like a empty is necesary